profiles/mercury.raml

```diff
@@ -167,7 +167,19 @@ validations:
       code: |
         function(resource) {
           const baseUriRegex = /^https:\/\/{shortCode}.api.commercecloud.salesforce.com\/([a-z]+-?)+\/([a-z]+-?)+\/{version}\/?$/;
-          const server = resource['apiContract:server'];
-          return server ? baseUriRegex.test(server[0]['core:urlTemplate'][0]) : false;
+          const servers = resource['apiContract:server'];
+          if(servers.length < 1) {
+            return false;
+          }
+          let result = true;
+          servers.forEach(server => {
```

Comment:
Is there a possibility of more than one server?

Reply:
I don't think so. Made this change just to make Jeff happy 😛

Reply:
can I suggest to check for >1 as well. probably this is a separate rule that executes before this one

Reply:
I like the idea of a separate rule for this. I will implement it in the next PR.